File: lista_2/overflow.py
limite = int(input())
num1,operador,num2 = map(str, input().split())
if operador == '+':
    resultado = int(num1) + int(num2)
elif operador == '*':
    resultado = int(num1) * int(num2)
if resultado <= limite:
    print('OK')
else:
    print('OVERFLOW')

File: lista_2/overflow.rb
limite = gets.chomp.to_i
num1, operador, num2 = gets.split.map(&:to_s)

if (operador == "+")
    resultado = num1.to_i + num2.to_i
    
elsif (operador == "*")
    resultado = num1.to_i * num2.to_i

end

if (resultado <= limite)
    puts "OK"
    
else
    puts "OVERFLOW"
    
end
